src/api/SearchApi.js

```diff
@@ -861,3 +862,33 @@ export function searchEntityNeighbors(entitySetId :UUID, entityId :UUID) :Promis
       return Promise.reject(error);
     });
 }
+
+
+/**
+ * `POST /search/{entityTypeId}/neighbors`
+ *
+ * Executes a search for all neighbors of all the entities specified, where all entities belong to the
+ * same entity set.
+ *
+ * @static
+ * @memberof lattice.SearchApi
+ * @param {UUID} entitySetId
+ * @param {UUID[]} entityIds
+ * @returns {Promise}
+ *
+ * @example
+ * SearchApi.searchEntityNeighborsBulk("ec6865e6-e60e-424b-a071-6a9c1603d735",
+ * ["3bf2a30d-fda0-4389-a1e6-8546b230efad","a62a47fe-e6a7-4536-85f1-fe935902a536"]);
+ */
+export function searchEntityNeighborsBulk(entitySetId :UUID, entityIds :UUID[]) :Promise<> {
+
+  return getApiAxiosInstance(SEARCH_API)
```

Comment:
Please add validation checks. These are easy since the parameters are just `UUID`s. You'll need `isValidUuid` and `isValidUuidArray`; refer to other places in this file for the same validation checks.